src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      return (this.result += value);
+    },
+
+    subtract(value) {
+      return (this.result -= value);
```

Comment:
[CODE STYLE] - Similar to the add method, the subtract method should perform the operation and then return the calculator object for method chaining.